index.js

```diff
@@ -1,28 +1,8 @@
-var countdownDate = new Date("Apr 15, 2023 09:00:00").getTime();
-
 $(document).ready(function() {
 	onResize();	
 	
 	$("#img_fallenstar").animate({left: "-65px"}, 1000);
 	$("#img_library").animate({right: "-50px"}, 1000);
-	
-	updateCountdown();
-	$("#countdown").animate({opacity: 1}, 500);
-	
-	setInterval(updateCountdown, 1000);
-	function updateCountdown() {
-		const time = countdownDate - (new Date().getTime());
-		const seconds = Math.floor(time/1000);
-		const minutes = Math.floor(seconds/60);
-		const hours = Math.floor(minutes/60);
-		const days = Math.floor(hours/24);
-		var timeStr = String(days).padStart(2, '0') + ":" + String(hours%24).padStart(2, '0') + ":" + String(minutes%60).padStart(2, '0') + ":" + String(seconds%60).padStart(2, '0');
-		timeStr = timeStr.split('').map(c => {
-			const span = `<span>${c}</span>`;
-			return span;
-		}).join('');
-		$("#countdown").html(timeStr);
-	}

```

Comment:
I think it would be nice to keep this functionality but just commented out. In the future we may want to implement a countdown when the event gets closer to the date.